package.json

```diff
@@ -25,7 +25,8 @@
     "tree-sitter-go": "^0.16.0",
     "tree-sitter-java": "^0.16.0",
     "tree-sitter-javascript": "^0.16.0",
-    "tree-sitter-python": "^0.16.0"
+    "tree-sitter-python": "^0.16.0",
+    "tree-sitter-typescript": "latest"
```

Comment:
What do you think about using the same version specification as for our other dependencies? 🙂